Initium-Odp/war/odp/javascript/script.js

```diff
@@ -1344,14 +1344,14 @@ function storeEnabled()
 	location.href = "ServletCharacterControl?type=storeEnabled"+"&v="+window.verifyCode;
 }
 
-function partyEnableJoins()
+function togglePartyJoins(eventObject)
 {
-	location.href = "ServletCharacterControl?type=enablePartyJoins"+"&v="+window.verifyCode;
-}
-
-function partyDisableJoins()
-{
-	location.href = "ServletCharacterControl?type=disablePartyJoins"+"&v="+window.verifyCode;
+	var clickedElement = $(eventObject.currentTarget);
```

Comment:
Using the updateHtml() concept, the whole clickedElement thing becomes obsolete.